docs/api-docs/cmds.md

```diff
@@ -1,206 +1,19 @@
-## `liftState(state): [any, Cmd]`
-
-* `state: any` &ndash; an object which may be the state of the redux store, or
-  an existing `[any, Cmd]` pair created by `loop()`.
-
-#### Notes
-
-Automatically converts objects to `loop()` results. If the value was created
-with `loop()`, then the function behaves as an identity. Otherwise, it is lifted
-into a `[any, Cmd]` pair where the effect is `Cmd.none()`. Useful for
-forcing reducers to always return a `loop()` result, even if they shortcut to
-just the model internally.
-
-#### Example
-
-```js
-function reducer(state, action) {
-  switch(action.type) {
-    case 'LOAD_START':
-      return loop(
-        { ...state, isLoading: true },
-        Cmd.run(apiFetch, {
-           successActionCreator: resolveActionCreator,
-           failActionCreator: rejectActionCreator,
-           args: [action.payload.id]
-        })
-    case 'LOAD_COMPLETE':
-      return {
-        ...state,
-        isLoading: false,
-        result: action.payload,
-      };
-    default:
-      return state;
-  }
-}
-
-// This guarantees that the return value of the reducer will be a loop result,
-// regardless of if it was set as such in the reducer implementation. This makes
-// it much easier to manually compose reducers without cluttering reducer
-// implementations with `loop(state, Cmd.none())`.
-export default compose(reducer, liftState);
-```
-
-
-## `getModel(loop): any`
-
-* `loop: any` &ndash; any object.
-* returns the model component of the array if the input is a `[any, Cmd]`
-  pair, otherwise returns the input object.
-
-#### Notes
-
-`getModel` lets you extract just the model component of an array returned by
-`loop`. It's useful in testing if you need to extract out the model component
-to do custom comparisons like `Immutable.is()`.
-
-
-## `getCmd(loop): Cmd | null`
-
-* `loop: any` &ndash; any object.
-* returns the cmd component of the array if the input is a `[any, cCmd]`
-  pair, otherwise returns `null`.
-
-#### Notes
-
-`getCmd` lets you extract just the cmd component of an array returned by
-`loop`. It's useful in testing if you need to separate the model and cmd and
-test them separately.
-
-
-## `isLoop(object): boolean`
- 
- * `object: any` &ndash; any object.
- * returns whether the given object was created with the `loop` function.
- 
- #### Notes
- 
- `isLoop` lets you determine whether an object returned by a reducer includes an
- cmd. This function is useful for writing custom higher-order functionality on
- top of redux-loop's API, or for just writing your own combineReducers.
-
-
-
-## `Cmds`
-
-#### Notes
+# `Cmds`
 
 The `Cmd` object provides access to all of the functions you'll need to
 represent different kinds of cmds to redux-loop's cmd processor. Every
 cmd is a plain JavaScript object that simply describes to the store how to
 process it. Cmd are never executed in the reducer, leaving your reducer pure
 and testable.
 
+## Command objects
+
+These are things that can be loop'ed
```

Comment:
This part is a little confusing. Is this describing the result of Cmd.foo vs Cmd itself?

Reply:
if you make the change above (rewording the first paragraph) I think you might be able to just remove this part about Command objects